packages/react/src/Blankslate/Blankslate.tsx

```diff
@@ -58,14 +58,19 @@ function Description({children}: DescriptionProps) {
   return <p className={clsx('Blankslate-Description', classes.Description)}>{children}</p>
 }
 
-export type PrimaryActionProps = React.PropsWithChildren<{
-  href: string
-}>
+export type PrimaryActionProps =
+  | (React.PropsWithChildren<{
+      href?: never
+    }> &
+      ButtonProps)
+  | React.PropsWithChildren<{
+      href: string
+    }>
 
-function PrimaryAction({children, href}: PrimaryActionProps) {
+function PrimaryAction({children, href, ...props}: PrimaryActionProps) {
   return (
     <div className={clsx('Blankslate-Action', classes.Action)}>
-      <Button as="a" href={href} variant="primary">
+      <Button as={href ? 'a' : 'button'} href={href} variant="primary" {...props}>
```

Comment:
```suggestion
      <Button {...props} as={href ? 'a' : 'button'} href={href} variant="primary">
```

One small suggestion, when spreading props I think it's helpful that it's always placed beforehand otherwise we could end up in a spot where someone unintentionally is overriding a prop that we set (e.g. they could override variant when we don't want them to)